web/pgadmin/browser/server_groups/servers/static/js/server.js

```diff
@@ -331,6 +331,30 @@ define('pgadmin.node.server', [
 
           return false;
         },
+        /* Refresh the server */
+        refresh: function(args, i) {
+          let obj = this,
+            tree = pgBrowser.tree,
+            item = i || tree.selected(),
+            data = item && tree.itemData(item);
+          // Fetch connection status.
+          fetch_connection_status(obj, data, tree, item);
```

Comment:
We had discussed not to call this function on refresh and use the node endpoint instead. The fix will be generic and will apply to all nodes.